src/pages/workspaces/workspace/Workflows.js

```diff
@@ -20,6 +20,7 @@ import { forwardRefWithName, memoWithName, useCancellation, useOnMount } from 's
 import * as StateHistory from 'src/libs/state-history';
 import * as Style from 'src/libs/style';
 import * as Utils from 'src/libs/utils';
+import * as WorkspaceUtils from 'src/libs/workspace-utils';
```

Comment:
In general, I note it's pretty inconsistent as to whether you do this pattern or just import the specific method(s) needed. I guess I would tend to just import the specific method when there's only one method of `WorkspaceUtils` being used.

Reply:
I tried to stay consistent with existing code or otherwise keep the change small. In this case, for example, the existing code was using methods from the `Utils` namespace, so changing the namespace seemed like the smallest change.